src/graphql/utils/getPageInfo.ts

```diff
@@ -0,0 +1,18 @@
+import { DEFAULT_LIMIT } from '../config'
+
+export default function getPageInfo(
```

Comment:
I think we already have a type for `PageInfo` generated that we can use as a return type for this function.